sched/jobDefSerializer_test.go

```diff
@@ -0,0 +1,249 @@
+package sched
+
+import (
+	"testing"
+	//"github.com/leanovate/gopter"
+	//"github.com/leanovate/gopter/prop"
+	"fmt"
+	"time"
+	"github.com/leanovate/gopter"
+	"github.com/leanovate/gopter/prop"
+)
+
+const serializeType = BinarySerialize
+
+
+func Test_JobDefCompare(t *testing.T) {
+	origJobDef := makeSampleJobDefintion()
+
+	// validate comparison logic
+	if ok, msg := origJobDef.Equal(origJobDef); !ok {
+		panic(fmt.Sprintf("error: comparison logic doesn't work comparing an object against itself %s\n", msg))
```

Comment:
recommend failing test cases with t.Error("error message") or t.Errorf("formated error message")
https://golang.org/pkg/testing/#B.Errorf

This will mark the test case as failed, and allow other test cases to run.  I believe panic will stop all the tests from executing if it fails.